app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -308,6 +309,9 @@ class BrowserActivity : DuckDuckGoActivity() {
         val url = webView.url
         doAsync {
             viewModel.addBookmark(title, url)
+            uiThread {
```

Comment:
If you're already on the UI thread, can't you just toast outside of the `async`?